site/ru/blog/new-in-devtools-98/index.md

```diff
@@ -6,228 +6,289 @@ authors:
 
 <!-- For example, I [recorded a user flow](/docs/devtools/recorder/#record) on this [demo page](https://jec.fyi/demo/pup-slow-result) to load and click on the menu item. However, the loading of the menu items is slow (it takes 6 seconds). The [replay](/docs/devtools/recorder/#replay) of this user flow failed because it exceeds 5 seconds  (the default timeout). -->
+Например, я [записала поведение пользователя](/docs/devtools/recorder/#record) на этой [демо-странице](https://jec.fyi/demo/pup-slow-result) при загрузке и клике на пункт меню. Однако загрузка пунктов меню происходит медленно (6 секунд). Воспроизведение [replay](/docs/devtools/recorder/#replay) этой записи не удалось, поскольку она превышает 5 секунд (таймаут по умолчанию).
 
 <!-- We can use the new **Timeout** settings to fix this. Expand the step which we click on the menu item. [Edit the step](/docs/devtools/recorder/#edit-steps) by  **Add timeout** and set it to **6000** milliseconds (equal to 6s). -->
+Мы можем использовать новую настройку **Timeout** чтобы исправить это. Расширьте шаг ожидания нажатия на пункт меню. [При редактировании шага](/docs/devtools/recorder/#edit-steps) нажмите кнопку **Add timeout** и установите значение **6000** миллисекунд (что равно 6 секундам).
 
 <!-- Optionally, you can adjust the **Timeout** in the **Replay settings** for all the steps. Expand the **Replay settings** and edit the **Timeout** value.  -->
+По желанию вы можете настроить **Timeout** для всех шагов в меню **Replay settings**. Раскройте настройки **Replay settings** и измените значение в окне **Timeout**. 
  
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/y7RDpIp3pd2n6Vnxc5Du.png", alt="timeout settings for user flow recording", width="800", height="530" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/y7RDpIp3pd2n6Vnxc5Du.png", alt="Настройка тайм-аута для записи поведения пользователя", width="800", height="530" %}
 
-Chromium issue: [1257499](https://crbug.com/1257499)
+Задача в трекере Chromium: [1257499](https://crbug.com/1257499)
 
 
 <!-- ## Ensure your pages are cacheable with the Back/forward cache tab {: #bfcache } -->
+## Убедитесь, что ваши страницы кэшируются с помощью вкладки Возвратный кэш (Back/forward cache) {: #bfcache }
 
 <!-- [Back/forward cache (or bfcache)](https://web.dev/bfcache/) is a browser optimization that enables instant back and forward navigation.  -->
+[Возвратный кэш (или bfcache)](https://web.dev/bfcache/) это оптимизация браузера, которая обеспечивает мгновенную навигацию назад и вперед. 
 
 <!-- The new **Back/forward cache** tab can help you test your pages to ensure they're optimized for bfcache, and identify any issues that may be preventing them from being eligible. -->
+Новая вкладка **Возвратный кэш** (Back/forward cache) поможет протестировать ваши страницы, чтобы убедиться, что они оптимизированы для возвратного кэша или выявить проблемы, которые можно исправить.
```

Comment:
```suggestion
Новая вкладка **Возвратный кэш** (Back/forward cache) поможет протестировать ваши страницы, чтобы убедиться, что они оптимизированы для возвратного кэша или выявить проблемы, которые этому препятствуют.
```

Reply:
Yes, that's where it comes from

![CleanShot 2022-01-24 at 11 37 33](https://user-images.githubusercontent.com/13676514/150748902-a39e245d-d53a-4710-b9a9-8d1a8f077c4c.png)